src/helpers/dates/companiDates.js

```diff
@@ -30,22 +55,40 @@ const companiDateFactory = (inputDate) => {
     },
 
     // MANIPULATE
+    startOf(unit) {
+      return CompaniDateFactory(_date.startOf(unit));
+    },
+
+    endOf(unit) {
+      return CompaniDateFactory(_date.endOf(unit));
+    },
+
     diff(miscTypeOtherDate, unit = 'milliseconds', typeFloat = false) {
       const otherDate = exports._formatMiscToCompaniDate(miscTypeOtherDate);
       const floatDiff = _date.diff(otherDate, unit).as(unit);
 
       if (typeFloat) return floatDiff;
       return floatDiff > 0 ? Math.floor(floatDiff) : Math.ceil(floatDiff);
     },
+
+    add(amount) {
```

Comment:
je ne saisis pas trop quelle est la différence entre ce add là et celui de dans companiDuration

Reply:
Dans companiDuration on augmente une durée avec une durée pour obtenir une durée augmentée (ex : 75 heures + 1 jour = 99 heures. 
Ici décale une date dans le temps d'une certaine durée pour obtenir une autre date( ex : lundi 3 janvier a 16h27 et 30 secondes + 1 jour = mardi 4 janvier a 16h27 et 30 secondes).

Le nom de la methode est peut etre mal choisi. On pourrai dire "postpone" https://www.linguee.fr/anglais-francais/traduction/postpone.html
ou push back https://www.linguee.fr/anglais-francais/traduction/push+back+the+date.html